mlir-sycl/lib/Dialect/SYCL/Transforms/ConstantPropagation.cpp

```diff
@@ -55,29 +56,33 @@ class ConstantPropagationPass
   void runOnOperation() final;
 
 private:
-  /// Return a range with all of the constant arguments of \p op.
-  static auto
-  getConstantArgs(SYCLHostScheduleKernel op, ArrayRef<Type> argumentTypes,
-                  ArrayAttr argAttrs, SymbolTableCollection &symbolTable,
-                  polygeist::SYCLAccessorAnalysis &accessorAnalysis);
-
-  /// Return all of the constant implicit arguments of \p op.
+  /// Inserts in \p input all of the constant arguments of \p op.
+  static SmallVectorImpl<std::unique_ptr<ConstantArg>> &getConstantExplicitArgs(
+      SmallVectorImpl<std::unique_ptr<ConstantArg>> &constants,
+      SYCLHostScheduleKernel op, ArrayRef<Type> argumentTypes,
+      ArrayAttr argAttrs, SymbolTableCollection &symbolTable,
+      polygeist::SYCLAccessorAnalysis &accessorAnalysis);
+
+  /// Inserts in \p input all of the constant implicit arguments of \p op.
```

Comment:
Update comment, "op" and "input" aren't parameters

Reply:
`op` is, in fact, but `input` isn't